packages/main/src/CalendarHeader.js

```diff
@@ -53,6 +53,15 @@ const metadata = {
 			type: CalendarType,
 		},
 
+		/**
+		 * Stores information for month button in second calendar type
```

Comment:
Stores information for month button for secondary calendar type

Reply:
Done